tools/data/hmdb51/download_annotations.sh

```diff
@@ -0,0 +1,20 @@
+#! /usr/bin/bash env
+
+DATA_DIR="../../../data/hmdb51/annotations"
+
+if [[ ! -d "${DATA_DIR}" ]]; then
+  echo "${DATA_DIR} does not exist. Creating";
+  mkdir -p ${DATA_DIR}
+fi
+
+cd ${DATA_DIR}
+wget http://serre-lab.clps.brown.edu/wp-content/uploads/2013/10/test_train_splits.rar --no-check-certificate
+
+# sudo apt-get install unrar
+unrar x test_train_splits.rar
+rm test_train_splits.rar
```

Comment:
it is very likely that user's pc does not have unrar installed. as a consequence, the newly downloaded data will be deleted without decompression. ref https://stackoverflow.com/questions/3474526/stop-on-first-error

Reply:
use `set -e` to make bash exit the script if any command fails